lib/main.js

```diff
@@ -343,7 +343,12 @@ const Hydrogen = {
     }
   },
 
-  createResultBubble(editor: atom$TextEditor, code: string, row: number) {
+  createResultBubble(
+    editor: atom$TextEditor,
+    code: string,
+    row: number,
+    cellType: string
```

Comment:
Good idea will implement. What would you call something like this because its technically not a cell?

Reply:
Note: This is from `createResultBubble()` and its private version `_createResultBubble()`.

I have implemented code and cell type combined to make `codeBlock`, but I am hesitant to add row since it doesnt pertain to the `codeBlock` its just where you want to display the result